software/beelzebub.yml

```
@@ -0,0 +1,14 @@
+name: "beelzebub"
+website_url: "https://beelzebub-honeypot.com/"
+source_code_url: "https://github.com/mariocandela/beelzebub"
+description: "A secure low code honeypot framework, leveraging AI for System Virtualization."
```

```suggestion
description: "Secure low code honeypot framework, leveraging AI for System Virtualization."
```

Minor syntax standardization